src/schema/v2/me/collectArtistsMutation.ts

```diff
@@ -0,0 +1,63 @@
+import { GraphQLList, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { GraphQLNonNull } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { meType } from "./index"
+
+interface Input {
+  artistIDs: string[]
+  userId: string
+}
+
+export const collectArtistsMutation = mutationWithClientMutationId<
+  Input,
+  any | null,
+  ResolverContext
+>({
+  name: "CollectArtistsMutation",
+  description: "Collect Multiple Artists",
+  inputFields: {
+    artistIDs: { type: new GraphQLNonNull(new GraphQLList(GraphQLString)) },
+    userId: { type: new GraphQLNonNull(GraphQLString) },
```

Comment:
**Question:** can we infer the user id from the session instead of sending it with the request?

Reply:
From session inside a mutation body?
How do you do that?